src/Controller/EnMarche/ProcurationController.php

```diff
@@ -99,18 +97,21 @@ public function profileAction(Request $request): Response
      */
     public function electionsAction(Request $request): Response
     {
-        $command = $this->getProcurationFlow()->getCurrentModel();
+        $procurationSession = $this->get(ProcurationRequestSession::class);
+        $command = $procurationSession->getCurrentModel();
```

Comment:
let's rename this variable to `$procuration` no ?

Reply:
I've renamed it `$procurationRequest` thanks.